### PR TITLE
Changed UI of Roadmap Page

### PR DESCRIPTION
**Pull request by khush2706**
## Description

I have changed the UI and layout of the Roadmaps page. I have changed the UI of project cards as well as of the modal wrapper.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue).

## Additional Context (Please include any Screenshots/gifs if relevant)
image

image

image

image

image

image

image

...

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by khush2706** [comment]
@HarshCasper can you please review my pull request.

**Comment by sarthakkundra** [comment]
Resolved the conflicts @khush2706 @HarshCasper  :)

File: website/src/pages/ProjectCard/ProjectCard.js [hunk 1]
```diff
@@ -6,52 +6,337 @@ import { motion, AnimatePresence } from 'framer-motion';
...
+                      </div>
+                    </div>
+                  </div>
+                </li>
+              </ul>
+            </div>
           </div>
         </div>
-        <div className="modal_links">
+        {/* <div className="modal_links">
```
Comment (sarthakkundra) [inline website/src/pages/ProjectCard/ProjectCard.js:325]: remove the commented out code if not required

File: website/src/pages/ProjectCard/card.css [hunk 2]
```diff
@@ -1,52 +1,74 @@
 .card {
-  width: 500px;
-  height: 250px;
-  margin-bottom: 10%;
-  margin-left: 15%;
+  /* width: 500px;
```
Comment (sarthakkundra) [inline website/src/pages/ProjectCard/card.css:2]: we have classes for this in tailwind CSS. Please use that. Also remove the commented out code

File: website/src/pages/ProjectCard/card.css [hunk 3]
```diff
@@ -60,41 +82,279 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
+
 .status h5 {
   text-align: center;
-  text-decoration: underline;
+  /* text-decoration: underline;
```
Comment (sarthakkundra) [inline website/src/pages/ProjectCard/card.css:148]: remove comment. Use Tailwind

**Review by sarthakkundra — CHANGES_REQUESTED** [review]

**Comment by sarthakkundra** [comment]
@khush2706 I see a lot of new CSS being written. While the UI looks good, I think we can achieve the same result with Tailwind CSS as well which is already configured in the project. This will also boost the performance of the website since the overall bundle size would be lesser. I'll appreciate it if you can do that. Thank you for contributing to moja global 💯

**Comment by khush2706** [comment]
@sarthakkundra I've made the suggested changes, can you pls review my PR again...also, there are a few things I need help with, I've messaged you on slack regarding the same :)

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hi @khush2706 , this PR looks great, awesome work. I have made few refactors in the code, you can have a look. Also, I have few suggestions that you can work upon this.

In the `ProjectCard.js` file, there are a lot of different components being used and are repetitive. We can create small components for them and use them in the ProjectCard.

Moreover, `handleClick` function uses `document.querySelector` DOM Api. In React Apps, it is not a good practice to directly change state and this may cause side effects. Can you come up with a `state`-based solution for handling this? We can brainstorm upon this if you want. Let me know if I can help anywhere in this.

**Comment by khush2706** [comment]
> Hi @khush2706 , this PR looks great, awesome work. I have made few refactors in the code, you can have a look. Also, I have few suggestions that you can work upon this.
> 
> In the `ProjectCard.js` file, there are a lot of different components being used and are repetitive. We can create small components for them and use them in the ProjectCard.
> 
> Moreover, `handleClick` function uses `document.querySelector` DOM Api. In React Apps, it is not a good practice to directly change state and this may cause side effects. Can you come up with a `state`-based solution for handling this? We can brainstorm upon this if you want. Let me know if I can help anywhere in this.

Sure! I'll make components for the timeline divs. As for the state-based solution, I'll try and see if I could make it work and I'll make sure to ask you if I get stuck anywhere.

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@sohamsshah Let's follow up with the refactor part after this PR gets merged. This is way too awesome work @khush2706 and I'm glad that you pulled this off! Thanks a lot for your contributions 🎉

**Pull request merged**